templates/digital-asset-template/frontend/tailwind.config.js

```diff
@@ -71,7 +73,112 @@ module.exports = {
         "accordion-down": "accordion-down 0.2s ease-out",
         "accordion-up": "accordion-up 0.2s ease-out",
       },
+      fontSize: {
+        12: "0.75rem",
+        14: "0.875rem",
+        16: "1rem",
+        18: "1.125rem",
+        20: "1.25rem",
+        24: "1.5rem",
+        26: "1.625rem",
+        30: "1.875rem",
+        39: "2.4375rem",
+        48: "3rem",
+        51: "3.1875rem",
+        68: "4.25rem",
+        110: "6.875rem",
+      },
+      lineHeight: {
+        16: "1rem",
+        18: "1.125rem",
+        20: "1.25rem",
+        22: "1.375rem",
+        24: "1.5rem",
+        26: "1.625rem",
+        32: "2rem",
+        34: "2.125rem",
+        42: "2.625rem",
+        52: "3.25rem",
+        62: "3.875rem",
+        72: "4.5rem",
+        116: "7.25rem",
+      },
     },
   },
-  plugins: [require("tailwindcss-animate")],
-}
\ No newline at end of file
+  plugins: [
+    require("tailwindcss-animate"),
+
+    plugin(function addTextStyles({ addComponents, theme }) {
+      addComponents({
+        // Component Regular Text Styles
+        ".component-100": {
+          fontFamily: theme("fontFamily.landing"),
+          fontSize: theme("fontSize.16"),
+          lineHeight: theme("lineHeight.28"),
+          fontWeight: theme("fontWeight.regular"),
+        },
+        ".component-200": {
```

Comment:
We should probably use t-shirt sizing in the typography names instead of `100`, `200`, etc. This naming scheme was a vestige of the V1 Aptos design system, but we have since switched to naming like `body-sm`, `body-md`, `body-lg`, etc. Tailwind uses t-shirt sizing throughout its theme system and the numeric size scale can be easily conflated with `font-weight` which uses similar values.

Reply:
Will change this